tests/test_ioc_scan.py

```diff
@@ -1,17 +1,18 @@
 #!/usr/bin/env pytest -vs
 """Tests for ioc_scan."""
 
+# Standard Python Libraries
 from io import StringIO
-from unittest.mock import patch
 import logging
 import sys
+from unittest.mock import patch
 
+# Third-Party Libraries
 import pytest
 
+# cisagov Libraries
 import ioc_scan
```

Comment:
Is it necessary to import both `ioc_scan` and specific components of it?

Reply:
The `ioc_scan` package doesn't export those component so the `import ioc_scan` is just to get `ioc_scan.__version__`. We need to import the specific components to access them for testing.

Reply:
Roger that!